.github/workflows/update-badge.yml

```diff
@@ -0,0 +1,57 @@
+name: Update Badge
+
+on:
+  workflow_run:
+    workflows:
+      - Dashboard
+    types:
+      - completed
+      - completed-with-failure
+    branches:
+      - master
+
+defaults:
+  run:
+    shell: bash
+
+concurrency:
+  group: ${{ github.workflow }}
+  cancel-in-progress: true
+
+jobs:
+  update-badge:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: pl-strflt/job-summary-url-action@v1
+        name: metadata
+        with:
+          workflow: dashboard.yml # ${{ github.event.workflow.path }}
+          run_id: ${{ github.event.workflow_run.id }}
+          run_attempt: ${{ github.event.workflow_run.run_attempt }}
+          job: generate-dashboard
+      - uses: actions/checkout@v3
+      # https://github.com/orgs/community/discussions/26560
+      - run: |
+          git config user.email "41898282+github-actions[bot]@users.noreply.github.com"
+          git config user.name "github-actions[bot]"
+      - run: |
+          GITHUB_JOB_SUMMARY_URL="${{ steps.metadata.outputs.job_summary_url }}"
```

Comment:
@galargh could you give me a hand with using the job summary action?

This variable is always empty at the moment (example run: https://github.com/laurentsenta/test-plans/actions/runs/3620111366/jobs/6101954513)

Reply:
Sure thing! There you go: https://github.com/libp2p/test-plans/pull/88/files/6b6007c2cf2c5496f739707c7c99ae290ed98547#r1039570057